src/manifest/clog.sidecar

```diff
@@ -10173,6 +10173,118 @@
         }
       ],
       "macroName": "QuicTraceLogInfo"
+    },
+    "RttUpdatedMsg": {
```

Comment:
Can you "clean" update to remove the old ones that no longer are used?

Reply:
we shouldnt, at least not in theory - the idea there is that the list grows